Bump classgraph dependency

I bumped into the same issue as mentioned here: https://github.com/karatelabs/karate/issues/2627 when trying to generate generate openapi specs in my Quarkus application.

```
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.Collection.iterator()" because the return value of "karate.nonapi.io.github.classgraph.reflection.ReflectionUtils.getFieldVal(boolean, Object, String)" is null 
at karate.nonapi.io.github.classgraph.classloaderhandler.QuarkusClassLoaderHandler.findClasspathOrderForQuarkusClassloader(QuarkusClassLoaderHandler.java:120) 
at karate.nonapi.io.github.classgraph.classloaderhandler.QuarkusClassLoaderHandler.findClasspathOrder(QuarkusClassLoaderHandler.java:111) at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
 ... 18 more
```

I was able to solve this by overriding the `io.github.classgraph:classgraph` in my pom:

```
<dependency>
    <groupId>io.swagger.core.v3</groupId>
    <artifactId>swagger-jaxrs2-jakarta</artifactId>
</dependency>
<!-- Override classgraph version from swagger-jaxrs2-jakarta -->
<dependency>
    <groupId>io.github.classgraph</groupId>
    <artifactId>classgraph</artifactId>
    <version>4.8.179</version>
</dependency>
```

Would it be possible to bump the classgraph dependency?

--- a/modules/swagger-project-jakarta/pom.xml
+++ b/modules/swagger-project-jakarta/pom.xml
@@ -474,7 +474,7 @@
         <jackson-version>2.19.2</jackson-version>
         <jackson-databind-version>2.19.2</jackson-databind-version>
         <logback-version>1.5.16</logback-version>
-        <classgraph-version>4.8.181</classgraph-version>
+        <classgraph-version>4.8.184</classgraph-version>
         <guava-version>32.1.3-jre</guava-version>
         <ant-version>1.10.15</ant-version>
         <commons-lang-version>3.18.0</commons-lang-version>

--- a/pom.xml
+++ b/pom.xml
@@ -631,7 +631,7 @@
         <jackson-version>2.19.2</jackson-version>
         <jackson-databind-version>2.19.2</jackson-databind-version>
         <logback-version>1.5.16</logback-version>
-        <classgraph-version>4.8.181</classgraph-version>
+        <classgraph-version>4.8.184</classgraph-version>
         <guava-version>32.1.3-jre</guava-version>
         <ant-version>1.10.15</ant-version>
         <commons-lang-version>3.18.0</commons-lang-version>

